src/components/views/messages/ReactionsRow.tsx

```diff
@@ -198,7 +198,8 @@ export default class ReactionsRow extends React.PureComponent<IProps, IState> {
         const cli = this.context;
 
         let addReactionButton;
-        if (cli.getRoom(mxEvent.getRoomId()).currentState.maySendEvent(EventType.Reaction, cli.getUserId())) {
+        const room = cli.getRoom(mxEvent.getRoomId());
+        if (room.getMyMembership() === "join" && room.currentState.maySendEvent(EventType.Reaction, cli.getUserId())) {
```

Comment:
some days I wonder if "may send event" should also check for membership...

Reply:
I thought that too but its jsdoc spoke only about permissions and I didn't want to trigger a js-sdk minor bump